boards/arm/mimxrt1170_evk/mimxrt1170_evk_cm4.dts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2021, NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <nxp/nxp_rt1170_cm4.dtsi>
+
+/ {
+	model = "NXP MIMXRT1170-EVK board";
+	compatible = "nxp,mimxrt1176";
+
+	aliases {
+		led0 = &green_led;
+		sw0 = &user_button;
```

Comment:
You can move the leds and buttons to a common dtsi, then include it in the cm4 and cm7 dts files.

See `boards/arm/lpcxpresso55s69/lpcxpresso55s69.dtsi` for example.

Reply:
Thank you for reminding me. I have modified.